ui-frontend/packages/catalog-ui-search/src/main/webapp/component/layer-item/layer-item.collection.view.js

```diff
@@ -23,7 +23,7 @@ import LayerItem from '../../react-component/layer-item'
 const LayerItemView = Marionette.ItemView.extend({
   attributes() {
     return {
-      'data-id': 'layer-item-collection-container',
+      'data-id': this.model.id,
```

Comment:
Not sure if this should be something like `${this.model.id}-layer-item-collection-container` or not.

Reply:
Probably should. We'll be using this for test cafe so we'll want something to indicate what it is.